js/parts-more/WaterfallSeries.js

```diff
@@ -241,6 +286,22 @@ seriesType('waterfall', 'column', {
                         }
                     }
 
+                    if (actualStackX.hasOwnProperty('absolutePos')) {
+                        if (stacking === 'normal' && !actualStackX.posTotal) {
+                            actualStackX.posTotal = actualStackX.absolutePos;
+                        }
+
+                        delete actualStackX.absolutePos;
+                    }
+
+                    if (actualStackX.hasOwnProperty('absoluteNeg')) {
+                        if (stacking === 'normal' && !actualStackX.negTotal) {
+                            actualStackX.negTotal = actualStackX.absoluteNeg;
+                        }
+
+                        delete actualStackX.absoluteNeg;
+                    }
+
```

Comment:
We have:
```
if (A)
if(B & C)

if (D)
(if B & E)
```
I think that better is:
```
if (B)
  if (A&C)

  if (D&E)
```

Reply:
I could not use the proposed scheme because delete actions also need to be performed when hasOwnProperty return true and stacking is set to 'overlap'.